Tasks/Common/azure-blobstorage-artifactProvider/azureBlobStorageProvider.ts

```diff
@@ -49,7 +49,7 @@ export class AzureBlobProvider implements models.IArtifactProvider {
     }
 
     public getRootItems(): Promise<models.ArtifactItem[]> {
-        return this._getItems(this._container)
+        return this._getItems(this._container, this._prefixFolderPath);
```

Comment:
does this change need porting to 123 branch

Reply:
no this code is not in M123.